ulmo/runs/SSL/llc_ssl_kin/llc_ssl_kin.py

```diff
@@ -0,0 +1,257 @@
+from __future__ import print_function
+
+import time
+import os
+import h5py
+import numpy as np
+import pandas as pd
+from tqdm.auto import trange
+import argparse
+
+
+from comet_ml import Experiment
+import torch
+
+#from ulmo import io as ulmo_io
+#from ulmo.llc import extract 
+
+from ulmo.ssl.util import adjust_learning_rate
+from ulmo.ssl.util import set_optimizer, save_model
+
+from ulmo.ssl.train_util import Params, option_preprocess
+from ulmo.ssl.train_util import modis_loader_v2, set_model
+from ulmo.ssl.train_util import train_model
+
+#from ulmo.utils import catalog as cat_utils
+
+#from ulmo.ssl import analysis as ssl_analysis
+
+#from IPython import embed
+
+def parse_option():
+    """
+    This is a function used to parse the arguments in the training.
+    
+    Returns:
+        args: (dict) dictionary of the arguments.
+    """
+    parser = argparse.ArgumentParser("argument for training.")
+    parser.add_argument("--opt_path", type=str, help="path of 'opt.json' file.")
+    parser.add_argument("--func_flag", type=str, 
+                        help="flags of the function to be execute: 'extract_curl', 'train', 'evaluate', 'umap'")
+    args = parser.parse_args()
+    
+    return args
+
+def main_train(opt_path: str):
+    # loading parameters json file
+    opt = Params(opt_path)
+    opt = option_preprocess(opt)
+
+    # build data loader
+    train_loader = modis_loader_v2(opt)
+
+    # build model and criterion
+    model, criterion = set_model(opt, cuda_use=opt.cuda_use)
+
+    # build optimizer
+    optimizer = set_optimizer(opt, model)
+    
+    # read 'user' and 'pin' for comet log
+    with open('/etc/comet-pin-volume/username', 'r') as f:
```

Comment:
what is `comet`?

Going forth, let's use the `main_train` code in the `modis_ssl_full` branch

Reply:
Comet is used to show the training process, namely the learning curve. We comment out the codes related to the comet in the main file.